serverless/messaging/reference-content/migration.mdx

```diff
@@ -0,0 +1,173 @@
+
+## Migration: timescales and action to take
+
+Below we explain for each protocol how the migration from the old to the new Messaging and Queuing will work, and what action you will need to take.
+
+### NATS
+
+Scaleway will automatically migrate each of your NATS namespaces and its credentials from the old version of Messaging and Queuing to the new one. In the new version, each namespace will effectively be a NATS account (each with its corresponding credentials). 
+
+The migration should be seamless: if you log into the Scaleway console before migration/GA you will see the "old" version of Messaging and Queuing with your existing resources showing as namespaces. After migration/GA, you will see the "new" version with your newly migrated resources now showing as accounts.
+
+In terms of the API, the [current `v1alpha1`](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) version will continue to be available 30 days after migration/General Availability, co-existing with the new `v1beta1` version. The documentation for both will be accessible via the [Scaleway Developers website](https://www.scaleway.com/en/developers/api) via a dropdown at the top of the Messaging and Queuing API page. 
+
+Internally, NATS namespaces (and their credentials) and NATS accounts (and their credentials) are the same resources, visible through both APIs. During the migration period, both APIs' endpoints are essentially accessing the same resources. Do not attempt to delete your NATS resources from the `v1alpha1` API as this will also delete them from the new `v1beta1` API and console. You will not be billed "twice" for resources in the old and new API, because these are essentially the same resources.
+
+Note that **the URL/endpoint by which you access your NATS resources will not be the same in the two versions**. By the end of the migration period, you **must** update any code, scripts, applications and architectures using the old endpoint, to update to the new endpoint. After this time, the old endpoints will no longer be recognized.
+
+The new URL/endpoint will be visible in the console after General Availability, and also from the new API.
+
+**Summary: NATS**
+
+✅ Migration from the old to new API will be automatically performed for you <br />
+✅ Billing of NATS resources will begin at the time of General Availability <br />
+✅ You must update your code, applications and architectures so they use the new endpoint no later than 30 days after General Availability
+
+### SQS
+
+From the time of Messaging and Queuing with SQS going into General Availability, you will have 30 days in which to migrate your SQS resources from the old version to the new version. You must migrate your resources yourself: this will not be automatic like it will for NATS.
```

Comment:
```suggestion
From the moment Messaging and Queuing with SQS goes into General Availability, you will have 30 days to migrate your SQS resources from the old to the new version. You must migrate your resources yourself: this will not be automatic like it will for NATS.
```